packages/front-end/components/Settings/UpgradeModal/index.tsx

```diff
@@ -549,43 +576,22 @@ export default function UpgradeModal({
                       <div className="row bg-main-color p-3 mb-3 rounded">
                         <span>You are currently using the </span>
                         <b className="mx-1"> {licensePlanText} </b> version of
-                        Growthbook with{" "}
-                        <Link
-                          href="/settings/team"
-                          className="mx-1 font-weight-bold"
-                        >
-                          {currentUsers} team members
-                        </Link>
-                        ↗
```

Comment:
The way we calculate seatsInUse between Stripe and Orb has diverged, to keep things simple, I've removed this.